website/source/docs/platform/k8s/run.html.md

```diff
@@ -17,14 +17,17 @@ This page starts with a large how-to section for various specific tasks.
 To learn more about the general architecture of Consul on Kubernetes, scroll
 down to the [architecture](/docs/platform/k8s/run.html#architecture) section. If you would like to get hands-on experience testing Consul on Kubernetes, try the step-by-step beginner tutorial with an accompanying video in the [Minikube with Consul guide](https://learn.hashicorp.com/consul/getting-started-k8s/minikube?utm_source=consul.io&utm_medium=docs)
 
-## Helm Chart
+## Helm Chart Installation
 
 The recommended way to run Consul on Kubernetes is via the
 [Helm chart](/docs/platform/k8s/helm.html). This will install and configure
 all the necessary components to run Consul. The configuration enables you
 to run just a server cluster, just a client cluster, or both. Using the Helm
 chart, you can have a full Consul deployment up and running in seconds.
```

Comment:
Not related to your change, but since I'm reading it, should this say "in minutes"? I feel like installing it in seconds is a bit of stretch.

Reply:
Seconds >60? 😆 I'll change it to a couple minutes